public/app/features/explore/state/history.ts

```diff
@@ -79,20 +77,31 @@ export const addHistoryItem = (
   queries: DataQuery[]
 ): ThunkResult<void> => {
   return async (dispatch, getState) => {
-    const { richHistoryStorageFull, limitExceeded } = await addToRichHistory(
-      datasourceUid,
-      datasourceName,
-      queries,
-      false,
-      '',
-      !getState().explore.richHistoryStorageFull,
-      !getState().explore.richHistoryLimitExceededWarningShown
-    );
-    if (richHistoryStorageFull) {
-      dispatch(richHistoryStorageFullAction());
-    }
-    if (limitExceeded) {
-      dispatch(richHistoryLimitExceededAction());
+    // always add to local storage
```

Comment:
The comment says exactly what the code, so maybe we can add a bit more info on "why"

```suggestion
    // always add to local storage - used to generate list of queries for autocomplete in Query Editors
```